src/main/java/pl/sknikod/kodemy/rest/response/MaterialShowGradesResponse.java

```diff
@@ -0,0 +1,52 @@
+package pl.sknikod.kodemy.rest.response;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import pl.sknikod.kodemy.grade.Grade;
+import pl.sknikod.kodemy.user.User;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class MaterialShowGradesResponse {
+    private Set<GradeResponse> grades;
```

Comment:
Z tego co pamiętam ustalaliśmy, że będziemy zwracać czysto listę, zatem w tym przypadku najlepiej będzie zrobić klasę `GradeResponse` jako podstawową z nazwą `SingleGradeResponse`, a w ResponseEntity zwracać `Set<SingleGradeResponse>`.
Unikniemy w takiej sytuacji pola o niepotrzebnej roli segregatora.
Oczywiście jeśli się mylę, to daj znać.